<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<arg name="action" default=""/>
	<arg name="sim_id" default="00"/> 
	<arg name="folder" default=""/> 

	<node name="launch_many_drones" pkg="rmader" type="launch_many_drones.py" 
		output="screen" clear_params="true" args="$(arg action) $(arg sim_id) $(arg folder)"/> 

</launch>


<!-- launch-prefix="xterm -e gdb -ex run args" -->